spec/migrations/umm_c_15_4_to_15_5_spec.rb

```diff
@@ -0,0 +1,205 @@
+require File.join(Rails.root, 'db', 'migrate', '20201221134953_umm_c_15_4_to_15_5')
+
+# Testing cases where: 
```

Comment:
I actually really like these comments, they helped to quickly/easily grasp was scenarios were being tested.